autoload/vundle/search.vim

```diff
@@ -0,0 +1,71 @@
+" ---------------------------------------------------------------------------
+" Search the database from vim-script.org for a matching plugin.  If no
```

Comment:
I suppose the comment needs updating now that we have embraced the github. the one true github. all hail our new overlords :alien: